server/reddit.js

```diff
@@ -0,0 +1,10 @@
+export default{
```

Comment:
Hi Grishma!
I was just reading this file and noticed that this export won't work. Node.js still doesn't allow native support for ES6 modules, so you'll have to use CommonJS modules (module.exports). At least on server-side code. For example:

```javascript
function search(search) {
  // ...
}

module.exports = { search }
// Or if you want it to be a default export:
module.exports = search;
```

Also, I noticed that you're not using the `search` parameter except for the `console.log`, and you have the query "burger" hardcoded.

Reply:
Hi John,
This is not my code. I merged him with reddit branch.

Reply:
Hi Grishma, sorry about that! I was just looking at Henry's code and thought "that looks familiar" 😆 .